React/Views/RCTShadowView.m

```diff
@@ -52,7 +52,7 @@ + (YGConfigRef)yogaConfig
 #if !TARGET_OS_OSX // [TODO(macOS GH#774)
     float pixelsInPoint = RCTScreenScale();
 #else
-    float pixelsInPoint = 0; // TODO(ISS#1656079): Turn off pixel rounding for macOS until we can get screen resolution passed here
+    float pixelsInPoint = 1; // TODO(ISS#1656079): Use 1x alignment for macOS until we can use backing resolution
```

Comment:
Thanks for looking at this. Can we just set this to the `_scale` ivar instead of hardcoding 1? It's a macOS-only prop on the shadow view

Reply:
Adding onto this, could we just reset the value at line 215? It looks like we can't use _scale here since it's a class property

Reply:
At Meta. we have applied the same patch about 1 year ago (internally).
I was actually thinking of upstreaming the change > https://github.com/christophpurrer/react-native-macos/commit/e50c0f3957ac4d1541f027e8a16016960b59b346